src/types/AdsConsent.interface.ts

```diff
@@ -127,3 +168,175 @@ export interface AdsConsentInfo {
    */
   isConsentFormAvailable: boolean;
 }
+
+/**
+ * The options used when requesting consent information.
+ */
+export interface AdsConsentUserChoices {
```

Comment:
Same concern as the other two spots, this seems like an area that will need semi-frequent refresh from source to get the consent correct as it changes over time. This one actually looks pretty duplicate too - is there a way to de-duplicate these three files so the same text is only in one spot? There may not be but it looks like a don't-repeat-yourself violation

Reply:
Im not sure how this works with TypeScript, editor resolution and the like. Wanted to have the description showing up during code completion as well as next to their definitions. Do you have a goof idea? Happy to change.

Reply:
No better ideas if you don't have any. Better too much doc than too little, no reason to hold up on this